obspy/core/stream.py

```diff
@@ -2564,13 +2564,36 @@ def normalize(self, global_max=False):
             tr.normalize(norm=norm)
         return self
 
-    def rotate(self, method, back_azimuth=None, inclination=None):
+    def rotate(self, method, back_azimuth=None, inclination=None,
+               inventory=None, **kwargs):
         """
         Rotate stream objects.
 
+        Example to rotate unaligned borehole instrument data based on station
+        inventory (a dataless SEED :class:`~obspy.io.xseed.parser.Parser` can
+        also be provided, see details for option ``inventory``):
+
+        >>> from obspy import read, read_inventory
+        >>> st = read("/path/to/ffbx_unrotated_gaps.mseed")
+        >>> inv = read_inventory("/path/to/ffbx.stationxml")
+        >>> st.rotate(method="->ZNE", inventory=inv)  # doctest: +ELLIPSIS
+        <obspy.core.stream.Stream object at 0x...>
+
```

Comment:
Would it be better to move this example to the bottom? And maybe add a couple other examples? Otherwise people might not realize that this can also do other kinds of rotations.